tests/dispatcher.spec.ts

```diff
@@ -0,0 +1,177 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import assert from "assert";
+import sinon from "sinon";
+
+import Dispatcher from "dispatcher";
+
+const sandbox = sinon.createSandbox();
+
+/**
+ * A sample task which returns a promise that takes between 0 and 10 ms to resolve.
+ *
+ * @returns The promise.
```

Comment:
nit: well, this could use some more explanation :D 
```suggestion
 * @returns The promise resolved when the timeout expires.
```

Reply:
Heh, for sure.